packages/engine/src/experiment/controller/run.rs

```diff
@@ -346,3 +349,36 @@ fn worker_pool_exit_logic(
 
     false
 }
+
+/// Shared memory cleanup in the process hard crash case.
+/// Not required for pod instances.
```

Comment:
Is this still relevant @Alfred-Mountfield?
```suggestion
///
/// Not required for pod instances.
```

Reply:
We don't have any k8s deployments or anything right now, so it's not

Reply:
Then lets remove that part.

Reply:
It is.. 😄 
<img width="527" alt="image" src="https://user-images.githubusercontent.com/25749103/164232418-293faae7-14ae-4068-a537-1adf5d805b30.png">